Build failure with DOCKER_HOST=tcp://...

### Description

Unable to build when environment variable `DOCKER_HOST` references **TCP** protocol

### Info

* d-m-p version : 0.33.0
* Maven version (`mvn -v`) :
```
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /usr/local/Cellar/maven/3.6.3_1/libexec
Java version: 11.0.7, vendor: Amazon.com Inc., runtime: /Library/Java/JavaVirtualMachines/amazon-corretto-11.jdk/Contents/Home
Default locale: en_CA, platform encoding: UTF-8
OS name: "mac os x", version: "10.14.6", arch: "x86_64", family: "mac"
```
* Docker version : 19.03.8
* running Docker Desktop on OS X 10.14 (therefore _no_ **docker-machine**)

#### Normal Use Case

- local build works fine in normal use-case: `mvn docker:build`
  i.e. there are _no_ `DOCKER_*` environment variables set at all

#### Local Kubernetes (K8s) Cluster Use Case

- trying to run local build by pointing Docker build to local [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) cluster, via:
  ```
  $ eval $(minikube docker-env)
  ```
  which results in setting these environment variables:
  ```
  $ export DOCKER_TLS_VERIFY="1"
  $ export DOCKER_HOST="tcp://127.0.0.1:32770"
  $ export DOCKER_CERT_PATH="/Users/alan/.minikube/certs"
  $ export MINIKUBE_ACTIVE_DOCKERD="minikube"
  ```
- local build with `docker build ./src/main/docker/` works fine
- local build with `mvn docker:build` fails with error:
  ```
  [ERROR] DOCKER> Unable to check image [repo.example.com:5001/docker-releases/base-image:1.0] : Client sent an HTTP request to an HTTPS server. (Bad Request: 400) [Client sent an HTTP request to an HTTPS server. (Bad Request: 400)]
  ```
- Dockerfile has:
  ```
  FROM repo.example.com:5001/docker-releases/base-image:1.0
  ...
  ```
- URL works fine from HTTPs: https://repo.example.com:5001/docker-releases/base-image:1.0
- if you change the `DOCKER_HOST` environment variable from **TCP** protocol to **HTTPs** protocol:
  ```
  $ export DOCKER_HOST="https://127.0.0.1:32770"
  ```
  then local build fails with: `unable to resolve docker endpoint: Invalid bind address format: https://127.0.0.1:32770`
  but then `mvn docker:build` works fine

##### Problem

Why is it that `docker build ./src/main/docker/` works fine but `mvn docker:build` fails when `DOCKER_HOST` environment variable uses **TCP** protocol? Why through the Maven Docker plugin it is trying HTTP instead of HTTPs but on Docker CLI it works fine?

##### Summary

No environment variables set:
- Maven Docker build works fine
- Docker CLI build works fine

`export DOCKER_HOST="tcp://127.0.0.1:32770"`
- Maven Docker build fails with: `Client sent an HTTP request to an HTTPS server. (Bad Request: 400)`
- Docker CLI build works fine

`export DOCKER_HOST="https://127.0.0.1:32770"`
- Maven Docker build works fine
- Docker CLI build fails with: `unable to resolve docker endpoint: Invalid bind address format: https://127.0.0.1:32770`

--- a/src/main/java/io/fabric8/maven/docker/util/EnvUtil.java
+++ b/src/main/java/io/fabric8/maven/docker/util/EnvUtil.java
@@ -31,17 +31,17 @@ public class EnvUtil {
 
     public static final String MAVEN_PROPERTY_REGEXP = "\\s*\\$\\{\\s*([^}]+)\\s*}\\s*$";
 
-    // Standard HTTPS port (IANA registered). The other 2375 with plain HTTP is used only in older
-    // docker installations.
-    public static final String DOCKER_HTTPS_PORT = "2376";
+    // Standard HTTPS port (IANA registered) is 2376.
+    // The other port 2375 with plain HTTP is used only in older docker installations.
+    public static final String DOCKER_HTTP_PORT = "2375";
 
     public static final String PROPERTY_COMBINE_POLICY_SUFFIX = "_combine";
 
     private EnvUtil() {}
 
-    // Convert docker host URL to an http URL
+    // Convert docker host URL to an HTTP(s) URL
     public static String convertTcpToHttpUrl(String connect) {
-        String protocol = connect.contains(":" + DOCKER_HTTPS_PORT) ? "https:" : "http:";
+        String protocol = connect.contains(":" + DOCKER_HTTP_PORT) ? "http:" : "https:";
         return connect.replaceFirst("^tcp:", protocol);
     }
 

